README.md

```diff
@@ -1 +1,166 @@
-# dask-pytorch
\ No newline at end of file
+# dask-pytorch
+
+<!-- ![GitHub Actions](https://github.com/saturncloud/dask-pytorch/workflows/GitHub%20Actions/badge.svg) [![PyPI Version](https://img.shields.io/pypi/v/prefect-saturn.svg)](https://pypi.org/project/prefect-saturn) -->
+
+`dask-pytorch` is a Python package that makes it easy to train PyTorch models on dask clusters using distributed data paralllel.  The intended scope of the project is
+- bootstrapping PyTorch workers on top of a Dask cluster
+- Implementations of common PyTorch datasets on distributed data stores (like S3)
+- mechanisms for tracking and logging intermediate results, training statistics, and checkpoints.
+
+## Typical non-dask workflow
+
+A typical example of non-dask PyTorch usage is as follows:
+
+### Loading Data
+Create an dataset (ImageFolder), and wrap it in a DataLoader
+
+```python
+transform = transforms.Compose([
+    transforms.Resize(256),
+    transforms.CenterCrop(250),
+    transforms.ToTensor()
+])
+
+whole_dataset = ImageFolder(path, transform=transform)
+
+batch_size = 100
+num_workers = 64
+indices = list(range(len(data)))
+np.random.shuffle(indices)
+train_idx = indices[:num]
+test_idx = indices[num:num+num]
+
+train_sampler = SubsetRandomSampler(train_idx)
+train_loader = DataLoader(data, sampler=train_sampler, batch_size=batch_size, num_workers=num_workers)
+```
+
+### Training a Model
+Loop over the dataset, and train the model by stepping the optimizer
```

Comment:
I notice you don't mention using a scheduler for the learning rate steps here, is that intentional to avoid complicating it?

Reply:
yes